roles/kubernetes-apps/external_provisioner/local_volume_provisioner/README.md

```diff
@@ -3,9 +3,47 @@ Local Storage Provisioner
 
 The local storage provisioner is NOT a dynamic storage provisioner as you would
 expect from a cloud provider. Instead, it simply creates PersistentVolumes for
-all manually created volumes located in the directories specified in the `local_volume_provisioner_storage_classes.host_dir` entries.
-The default path is /mnt/disks and the rest of this doc will use that path as
-an example.
+all manually created volumes specified in the `local_volume_provisioner_storage_classes` list.
```

Comment:
This could be more specific. Maybe clarify that mounts created on the host under the `host_dir` path of a specified storage class will be automatically added as PVs.

Reply:
Added a link to the local-provisoner Github and rewrote the sentences a bit. Does this resolve the issue?